pkgs/unified_analytics/lib/src/utils.dart

```diff
@@ -137,13 +137,17 @@ Directory? getHomeDirectory(FileSystem fs) {
 /// Dart: `$HOME/.dart/dartdev.json`
 ///
 /// Flutter: `$HOME/.flutter`
+///
+/// Devtools: `$HOME/.flutter-devtools/.devtools`
```

Comment:
So here "legacy" doesn't necessarily refer to the analytics instance it's going to, but an implementation of analytics that predates package:unified_analytics, right?

And therefore, we will need to maintain this code in perpetuity?

Reply:
Couldn't it mean both? Currently devtools has their own implementation for analytics while it also sends to a different GA instance.. is that what you mean?

And yes, we would maintain this. It only runs the first time `package:unified_analytics` is run on a developers workstation though